src/main/java/ca/wolfram/beta/symath/operations/MultiplyNode.java

```diff
@@ -45,7 +91,15 @@ public double operationEval(VMap map) {
     public String toString() {
         StringBuilder s = new StringBuilder();
         s.append("(");
-        getChildren().forEach((c) -> s.append(c).append(" * "));
-        return s.substring(0, s.length() - 3) + ")";
+        boolean isFirst = true;
+        for (MathNode c : getChildren()) {
+            if (isFirst) {
+                s.append(c.toString());
+                isFirst = false;
+                continue;
+            }
+            s.append(" * ").append(c.toString());
+        }
```

Comment:
Yea, but I feel like it is easy to prevent the use of `continue` and the use of the flag `isFirst`. Also my implementation is as easy to add a conversion thing for divide.